examples/summarization/finetune.sh

```diff
@@ -11,6 +11,8 @@ export PYTHONPATH="../":"${PYTHONPATH}"
 
 python finetune.py \
     --model_name_or_path=facebook/bart-large \
+    --data_dir=./cnn-dailymail/cnn_dm \
```

Comment:
Can we leave this in the `README` and add a comment in this file that the proper usage is documented in the README. I would rather not hardcode paths to data in this script, since people are using many different datasets stored in different places.

Reply:
Sure, please check my 2nd commit, i also dropped `cnn` prefix from default output dir in finetune.sh. Maybe it makes sense not to hardcode it as well? Will be glad to hear your feedback.